common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,126 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from django.contrib.auth.models import User
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+from requests_oauthlib import OAuth1Session
+
+log = logging.getLogger(__name__)
```

Comment:
I think this is going off of the existing mongodb backend: https://github.com/Stanford-Online/edx-platform/blob/master/common/djangoapps/track/backends/mongodb.py#L14

should change it there in a separate PR?